### ISSUE TITLE
Change the name of StripeButton in the example

### ISSUE THREAD

**Issue by dcaravana** [yariv/ReactScriptLoader#9]
Just a little advice: change that name to something like `ReactStripeButton` because `checkout.js` defines `StripeButton`, too, so when you build the React objects the first time all is fine (because stripe's script is loaded asynchronously), then if something happens that require to access again to the React object `StripeButton` was pointing to (like a `setState()` call), React throws exception (`Uncaught TypeError: type.toUpperCase is not a function`).

Really a little nasty bug, it took my hours to find and fix.

**Comment by yariv** [comment]
I'll be happy to make the change. Would you like to submit a diff?

**Comment by dcaravana** [comment]
Yep sure! Double check it since I didn't try (my own code already diverged too much from this).

https://gist.github.com/dcaravana/4d9c2903f2ca2afa7e7b

**Comment by dmitry** [comment]
BTW stripe module: https://github.com/azmenak/react-stripe-checkout
